taxi/admin.py

```diff
@@ -0,0 +1,32 @@
+from django.contrib import admin
+
+# Register your models here.
+from django.contrib import admin

```

Comment:
The import statement for `admin` is duplicated. You should remove one of these import statements to avoid redundancy.